credentials/alts/core/gen.go

```diff
@@ -0,0 +1,38 @@
+/*
+ *
+ * Copyright 2018 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+// Package core contains common core functionality for ALTS.
+//go:generate /bin/sh -exu -o pipefail -c "tail -n +22 ./gen.go | /bin/sh -exu -o pipefail"
```

Comment:
Is it possible to put this into a `.sh` file and run it instead?

Also, you can put the `go:generate` into an existing file, that should be fine.

Reply:
Yes, done.